x-pack/solutions/security/plugins/security_solution/common/api/model/alert.gen.ts

```diff
@@ -15,21 +15,29 @@
  */
 
 import { z } from '@kbn/zod';
-
-import { NonEmptyString } from './primitives.gen';
+import { isNonEmptyString } from '@kbn/zod-helpers';
 
 /**
- * A list of alerts ids.
+ * A list of alerts `id`s.
  */
 export type AlertIds = z.infer<typeof AlertIds>;
-export const AlertIds = z.array(NonEmptyString).min(1);
+export const AlertIds = z.array(z.string().min(1).superRefine(isNonEmptyString)).min(1);
```

Comment:
Let me know if there's a better way, but using ref to `NonEmptyString` didn't allow me to override a `description`.